src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltObservationTable.js

```diff
@@ -430,9 +432,16 @@ const FishBeltObservationTable = ({
         </StickyObservationTable>
       </StyledOverflowWrapper>
       <UnderTableRow>
-        <ButtonPrimary type="button" onClick={handleAddObservation}>
-          <IconPlus /> Add Row
-        </ButtonPrimary>
+        <UnderTableRowButtonArea>
+          <ButtonPrimary
+            type="button"
+            onClick={handleAddObservation}
+            disabled={!fishBinSelectedLabel}
+          >
+            <IconPlus /> Add Row
```

Comment:
Not suggesting you change this here since pushing more changes and retesting is a pain, but as we work through code moving forward, and we bump into old hard coded text, can we start adding it to the language file?

Reply:
noted